horizon/templates/local_settings.py.erb

```diff
@@ -261,7 +261,7 @@ API_RESULT_PAGE_SIZE = 20
 
 # The timezone of the server. This should correspond with the timezone
 # of your entire OpenStack installation, and hopefully be in UTC.
-TIME_ZONE = "UTC"
```

Comment:
yeah, I wasn't able to find a straight answer easily, but I found a few hints that this might be handled by django.  But, we are currently in Daylight Savings.

Ruoyu caught up to me in person and mentioned it is showing the correct time now in tests, and we're still in Daylight savings, so it probably works by itself then.  I guess we'll find out for sure next month!